dali/operators/reader/gds_mem.cc

```diff
@@ -12,40 +12,283 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <map>
 #include <memory>
+#include <mutex>
 #include <utility>
 #include <vector>
 #include "dali/operators/reader/gds_mem.h"
+#include "dali/util/cufile_helper.h"
+#include "dali/core/math_util.h"
 #include "dali/core/spinlock.h"
 #include "dali/core/mm/pool_resource.h"
+#include "dali/core/mm/detail/align.h"
 #include "dali/core/mm/malloc_resource.h"
 #include "dali/core/mm/composite_resource.h"
 
 namespace dali {
+namespace gds {
+
+static size_t GetGDSChunkSizeEnv() {
```

Comment:
It can be moved to anonymous namespace. Why make it static?

Reply:
I thought that `static` is more C style, while namespace is more C++. Up to you.